packages/components/organisms/f-checkout/src/components/ErrorDialog.vue

```diff
@@ -64,20 +65,35 @@ export default {
     },
 
     mounted () {
-        this.isOpen = true;
+        const modalContext = this.getModalContext();
+
+        if (modalContext) modalContext.open();
     },
 
     methods: {
         ...mapActions(VUEX_CHECKOUT_MODULE, [
             'updateMessage'
         ]),
 
+        getModalContext () {
+            const { errorModal } = this.$refs;
+            const { megaModal } = errorModal.$refs;
+
+            if (megaModal) return errorModal;
```

Comment:
Is this correct? It looks weird to me that we're checking if there is an `errorModal.$refs.megaModal` and if there is one, we return `errorModal`, but I don't know the context of this. Just to double check :)

Reply:
The megaModal reference is to make sure we have the element on the page, if we don't check that then we'd be closing/opening the modal without knowing if the modal is actually open or not. Let me know if that makes sense.

Reply:
It kinda does. I'll trust on you this one 😄

Reply:
Would be useful to add a JSDoc to this method, as think this explanation helps give some context (as agree it's not obvious from the code, but makes sense with your explanation).

Reply:
Will update.